torch/ao/quantization/fx/_decomposed.py

```diff
@@ -94,3 +94,32 @@ def choose_qparams_tensor(input, quant_min, quant_max, dtype):
     observer(input)
     scale, zero_point = observer.calculate_qparams()
     return (scale, zero_point)
+
+# Helper function used to implement per-channel quantization against any axis
+def _permute_to_axis_zero(x, axis):
+    new_axis_list = list(range(x.dim()))
+    new_axis_list[axis] = 0
+    new_axis_list[0] = axis
+    y = x.permute(tuple(new_axis_list))
+    return y, new_axis_list
+
+quantized_decomposed_lib.define(
+    "quantize_per_channel(Tensor input, Tensor scales, Tensor zero_points, int axis, "
+    "int quant_min, int quant_max, ScalarType dtype) -> Tensor")
+
+@impl(quantized_decomposed_lib, "quantize_per_channel", "CompositeExplicitAutograd")
+def quantize_per_channel(input, scales, zero_points, axis, quant_min, quant_max, dtype):
```

Comment:
do we plan to have any documentation for functions in this library?

Reply:
yeah, I can add some docs

Reply:
added this in a separate PR to avoid the CI wait time for this PR